Documentation/api.rst

```diff
@@ -82,15 +82,8 @@ The full example can be found in the `cilium/client-example`_ repository.
 Compatibility Guarantees
 ************************
 
-The Cilium API is still marked **unstable** overall at this point but several
-core sections have already matured a lot. We will declare the overall API
-stable with the release of version 1.0. In the meantime, we would like maintain
-the flexibility to adjust to feedback. The APIs are also heavily dependent on
-the evolution of various orchestration systems which have not stabilized yet.
-
-If you are planning to utilize the API and could benefit from a stable API,
-please approach us on Slack and we can discuss marking certain sections of the
-API stable and start maintaining backwards compatibility.
+Cilium API is stable as of version 1.0, backward compatibility will be upheld
+for whole lifecycle of 1.0.
```

Comment:
Does this mean 1.1 may be different? I thought we intended to keep it compatible for minor and patch releases but I might be wrong.
Separately, I was under the impression that we wouldn't remove or break existing fields but might add new ones. This won't break many JSON parsers but might break validation or tests that compare the json output as a string etc.
Separately separately, we marked `PUT /endpoint` as deprecated but I'm not sure it shows up anywhere. Should we address deprecation timelines here?

Reply:
I think it's bad wording on my part. I will change this so that it's clear that API is backward compatible throughout major version 1.